src/components/services-view/services-view.controller.ts

```diff
@@ -65,12 +65,17 @@ export class ServicesViewController implements angular.IController {
     }
 
     this.ctrl.currentFilter = category;
-    this.ctrl.currentSubFilter = (this.ctrl.subCategories.length === 1) ? this.ctrl.subCategories[0].id : (subCategory || 'all');
+    this.ctrl.currentSubFilter = (this.ctrl.subCategories.length === 1) ? this.ctrl.subCategories[0].id : (subCategory || null);
```

Comment:
@cshinn, @dtaylor113 The logic changes here break the functionality for categories that don't have a subcategory or more than one subcategory (All, CI/CD, Other).  You'll note clicking one of those either shows the items for All or the items from the last subcategory selected.  cc:  @spadgett